boxsdk/object/item.py

```diff
@@ -613,3 +621,94 @@ def get_collaborations(self, limit=None, marker=None, fields=None):
             fields=fields,
             return_full_pages=False,
         )
+
+    def add_classification(self, classification):
+        """
+        Applies metadata classification for the specified :class:`File` or :class:`Folder` object.
+
+        :param classification:
+            The classification to add to the :class:`File` or :class:`Folder`
+        :type classification:
+            `unicode`
+        :return:
+            The classification added to the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        classification_metadata = {
+            'Box__Security__Classification__Key': classification,
+        }
+        template_key = 'securityClassification-6VMVochwUWo'
```

Comment:
It might be worth pulling the template key and field key out into constants so you don't need to keep repeating them everywhere (and so they're easy to update in the future if we need to)